### PR TITLE
ref(location): optimize http request (using cache)

### PR DESCRIPTION
**Pull request by cmgustavo**
* Removed unused css classes
* Use Cached country instead to call same request

### CODE REVIEW

File: src/providers/tab/tab.ts [hunk 1]
```diff
@@ -9,26 +13,25 @@ export class TabProvider {
   activeGiftCardsPromise: Promise<GiftCard[]>;
```
Comment (JohnathanWhite) [inline src/providers/tab/tab.ts:13]: we can also remove the bitpayCardItemsPromise declaration here

Comment (cmgustavo) [reply src/providers/tab/tab.ts:13]: +1

**Comment by JohnathanWhite** [comment]
ACK!

**Review by gabrielbazan7 — APPROVED** [review]
ACK

**Pull request merged**